docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1118,15 +1118,38 @@ public class MyConfiguration {
 
 One type of bean that it may be useful to register using <<custom-loadbalancer-configuration,Custom LoadBalancer configuration>> is `LoadBalancerLifecycle`.
 
-The LoadBalancerLifecycle beans provide callback methods, named `onStart(Request<RC> request)` and `onComplete(CompletionContext<RES, T> completionContext)`, that you should implement to specify what actions should take place before and after load-balancing.
+The LoadBalancerLifecycle beans provide callback methods, named `onStart(Request<RC> request)`, `onStartRequest(Request<RC> request, Response<T> lbResponse)` and `onComplete(CompletionContext<RES, T, RC> completionContext)`, that you should implement to specify what actions should take place before and after load-balancing.
```

Comment:
Add backticks around `LoadBalancerLifecycle` (the second word in the sentence)

You didn't do it, but please fix it, because you (correctly) put it in backticks in your new content.